TwilioStore.Interfaces/Exceptions/InsufficientInventoryException.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using TwilioStore.Interfaces.Domain;
+
+namespace TwilioStore.Interfaces.Exceptions
```

Comment:
Not sure about having the `Exceptions` under the `Interfaces` namespace. To me those exceptions belong to services. Both are used over there.